river_dl/preproc_utils.py

```diff
@@ -860,7 +873,9 @@ def prep_all_data(
     x_data = x_data[x_vars]
 
     if catch_prop_file:
-        x_data = prep_catch_props(x_data, catch_prop_file, spatial_idx_name)
+        x_data = prep_catch_props(x_data, catch_prop_file, catch_prop_vars, spatial_idx_name)
+        #update the list of x_vars
+        x_vars = [i for i in x_data.data_vars]
```

Comment:
can you just do this?
```suggestion
        x_vars = list(x_data.data_vars)
```

Reply:
yep, that works.